metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/BaseJcrProvider.java

```diff
@@ -333,23 +338,70 @@ public T findFirst(String query) {
         }
     }
 
+    protected String getFindAllFilter() {
+        return getFindAllFilter(getEntityQueryStartingPath());
+    }
+
+    protected String getFindAllFilter(String startingPath) {
+        int depth = 0;
+        if (startingPath != null) {
+            depth = startingPath.split("/").length;
+        }
+        return getFindAllFilter(startingPath, depth);
+    }
+
+    protected String getFindAllFilter(String startingPath, int depth) {
+        boolean skip = false;
+        if (skip) {
+            return null;
+        }
+        if (startingPath == null) {
+            return null;
+        } else {
+            return "ISDESCENDANTNODE('" + startingPath + "') AND ( DEPTH() = CAST(" + depth + " AS LONG) ) "; // OR DEPTH() = CAST(" + (depth + 1) + " AS LONG) ) ";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)